src/docs/doxygen/mainpage.md

```diff
@@ -3,8 +3,8 @@
 #### Physics Simulation Modules ####
 
 * [Base physics](@ref serac::BasePhysics)
-* [Heat Transfer](@ref serac::HeatTransfer)
-* [Solid mechanics](@ref serac::SolidMechanics)
+* [Heat Transfer](classserac_1_1HeatTransfer_3_01order_00_01dim_00_01Parameters_3_01parameter__space_8_8_8_01_4_00b10147e2187ea0fff3ed4d400f86c2eb.html)
+* [Solid mechanics](classserac_1_1SolidMechanics_3_01order_00_01dim_00_01Parameters_3_01parameter__space_8_8_8_01_4_0a4b58643425b41433838b2828455c49.html)
```

Comment:
Is there a reason to remove the doxygen `@ref` commands? Is it not functioning, or was it the wrong thing to do in the first place? There's an incongruity between the unmodified links and the new explicit references to (very ugly) html files. Possibly related: when I build the documentation on my system, the links that are modified look different, being rendered in regular typeface instead of bold.

Reply:
`@ref` didn't seem to work when I tried initially, but you're right that the formatting is different-- presumably why the links in lower in the file resort to HTML rather than markdown link syntax to ensure things are rendered bold. I'll look again and see if I can make things stylistically consistent again.